patches/0013-p256-armv8-optimizations

```diff
@@ -2429,20 +2457,460 @@ Index: aws-lc/third_party/boringssl/crypto/fipsmodule/ec/p256-x86_64_test.cc
      !defined(OPENSSL_SMALL) && !defined(BORINGSSL_SHARED_LIBRARY)
  
  TEST(P256_X86_64Test, SelectW5) {
-@@ -98,6 +97,7 @@ TEST(P256_X86_64Test, SelectW7) {
-   CHECK_ABI(ecp_nistz256_select_w7, &val, table, 42);
+   // Fill a table with some garbage input.
+-  alignas(64) P256_POINT table[16];
++  __attribute__((aligned(64))) P256_POINT table[16];
+   for (size_t i = 0; i < 16; i++) {
+     OPENSSL_memset(table[i].X, 3 * i, sizeof(table[i].X));
+     OPENSSL_memset(table[i].Y, 3 * i + 1, sizeof(table[i].Y));
+@@ -71,7 +70,7 @@ TEST(P256_X86_64Test, SelectW5) {
+ 
+ TEST(P256_X86_64Test, SelectW7) {
+   // Fill a table with some garbage input.
+-  alignas(64) P256_POINT_AFFINE table[64];
++  __attribute__((aligned(64))) P256_POINT_AFFINE table[64];
+   for (size_t i = 0; i < 64; i++) {
+     OPENSSL_memset(table[i].X, 2 * i, sizeof(table[i].X));
+     OPENSSL_memset(table[i].Y, 2 * i + 1, sizeof(table[i].Y));
+@@ -99,11 +98,12 @@ TEST(P256_X86_64Test, SelectW7) {
  }
  
-+#if !defined(OPENSSL_AARCH64)
  TEST(P256_X86_64Test, BEEU) {
++#if !defined(OPENSSL_AARCH64)
```

Comment:
Why is this not relevant for ARM?

Reply:
This is what I alluded to in the response to the comment on l.601. The variable underneath this line is only defined for x86_64. In https://github.com/nebeid/aws-lc/tree/CA591-clean, I made it
`#if defined(OPENSSL_X86_64)`
I changed it here too.